src/app/problem/ProblemList.js

```diff
@@ -34,34 +35,86 @@ const ProblemList = () => {
         history.push(`/problems/${problemId}/edit`);
     };
 
-    if (!problems)
+    const ProblemFilterSearchBar = ({filterItems}) => {
+        return (
+            <div className="is-flex is-justify-content-center filterSearchBar">
+                <div className="select">
+                    <select>
+                        {filterItems?.map(name => <option key={name}>{name}</option>)}
+                    </select>
+                </div>
```

Comment:
I guess we don't need `<div>` here. 🤔 
Remove it if we don't need it.

Reply:
It has its own style in ProblemList.scss flie.